.evergreen/config.yml

```diff
@@ -4514,12 +4514,6 @@ buildvariants:
     tasks:
       - test_azurekms_task_group
       - test-azurekms-fail-task
-  - name: ubuntu20-test-azure-oidc
-    display_name: Azure OIDC
-    run_on: ubuntu2004-small
-    batchtime: 20160
-    tasks:
-      - testazureoidc_task_group
```

Comment:
Is there a reason a similar TODO(DRIVERS-2416/NODE4929) comment wasn't left in this file? Is it just assumed?